internal/backends/qt/qt_widgets.rs

```diff
@@ -174,6 +174,30 @@ cpp! {{
         // (because the QGuiApplication destructor access some Q_GLOBAL_STATIC which are already gone)
         new QApplication(argc, argv2);
     }
+
+
+    template <typename Base>
+    struct SlintAnimatedWidget: public Base {
+        void *animation_update_property_ptr;
+        bool event(QEvent *event) override {
+            if (event->type() == QEvent::StyleAnimationUpdate) {
+                rust!(Slint_AnimatedWidget_update [animation_update_property_ptr: Pin<&Property<i32>> as "void*"] {
+                    animation_update_property_ptr.set(animation_update_property_ptr.get() + 1);
```

Comment:
i guess this is to workaround the fact that Slint cashes the image, and would therefore not animate?
(and this invalidates the cache)
But i'm missing the location where we do the get() on this property to register the dependency.